.github/workflows/openms_ci_matrix_full.yml

```diff
@@ -288,9 +288,17 @@ jobs:
     - name: Download contrib build from archive (Windows)
       if: startsWith(matrix.os, 'windows') && steps.cache-contrib.outputs.cache-hit != 'true'
       shell: bash
+      env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: |
           cd OpenMS/contrib
-          curl --no-progress-meter -L -o contribbld.tar.gz https://abibuilder.cs.uni-tuebingen.de/archive/openms/contrib/${{ steps.set-vars.outputs.contrib_os }}/${{ steps.set-vars.outputs.contrib_os_ver }}x64/${{ steps.set-vars.outputs.contrib_compiler }}contrib_build.tar.gz
+          # Use GitHub API to get the download URL for the latest release asset named 'contrib_build-Windows.tar.gz'
+          DOWNLOAD_URL=$(curl -sH "Authorization: token $GITHUB_TOKEN" \
+            https://api.github.com/repos/t0mdavid-m/contrib/releases/latest \
```

Comment:
use OpenMS org repo

Reply:
Thank you! I completely forgot to change that.